src/app/routes/utils/fetchPageData/index.js

```diff
@@ -67,12 +67,19 @@ const handleError = e => {
   };
 };
 
-const fetchData = pathname => {
+const fetchData = (pathname, { maximumAttempts = 1 } = {}) => {
```

Comment:
I'd prefer it if we had a feature toggle for this, so we can turn it off across all of Simorgh if it's causing a problem. This would enable us to try it out on local/on Test for a period of time before trying this on the live environment.